tests/lib/rules/await-async-queries.test.ts

```diff
@@ -93,6 +93,9 @@ ruleTester.run(RULE_NAME, rule, {
 		// async screen queries declaration are valid
 		...createTestCase((query) => `await screen.${query}('foo')`),
 
+		// async queries with optional chaining are valid
```

Comment:
suggestion(blocking): This is nice, but please add also another test for the exact scenario reported in #882 description.

Reply:
Done